### PR TITLE
Home page improvement

### PR DESCRIPTION
**Pull request by Kyle199910**
Minor adjustment to the home page: text adjustment and lighter shape

### CODE REVIEW

**Comment by gaurav613** [comment]
This looks good to me. @Rolo123y can you do some testing to make sure everything works well?

**Review by Rolo123y — CHANGES_REQUESTED** [review]
Can you remove the app/__pycache__ stuff from the commits?

File: app/templates/FAQ.html [hunk 1]
```diff
@@ -1,64 +1,70 @@
 {% extends 'base.html'%} {% block title%} FAQ {% endblock%} {% block styles %}
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: {% block styles %} shouldnt be defined here as it is defined a bit under too.

File: app/templates/FAQ.html [hunk 2]
```diff
@@ -1,64 +1,70 @@
...
+{% block styles %}
+   
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/faq.css')}}">
+
+{% endblock %}
+
 
 <!-- CONTENT HERE -->
 
 <body>
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: replace this <body> tag with {% block content %}

File: app/templates/FAQ.html [hunk 3]
```diff
@@ -1,64 +1,70 @@
...
+        <label for="acc6">Modi excepturi</label>
+        <div class="content">
+            <div class="inner">
+                Iorem ipsum
             </div>
+        </div>
+    </div>
 </body>
 
 </html>
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: remove these tags.

**Review by Rolo123y — CHANGES_REQUESTED** [review]

**Review by Rolo123y — APPROVED** [review]
We can resolve the __init__.py changes

other than that everything looks good!

**Comment by gaurav613** [comment]
Changes look good - I have resolved the conflicts. Merging now.

**Pull request merged**